spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/startup/StartupTimeMetricsProperties.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2012-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.actuate.autoconfigure.metrics.startup;
+
+/**
+ * Nested configuration properties for startup time metrics.
+ *
+ * @author Chris Bono
+ * @since 2.6.0
+ */
+public class StartupTimeMetricsProperties {
+
+	/**
+	 * Name of the metric for time taken for the application to start.
+	 */
+	private String applicationStartedTimeMetricName = "application.started.time";
```

Comment:
I am really not in ❤️ w/ these uber long names (especially when you see them in use). Thoughts?

Reply:
On 2nd thought...

@philwebb wdyt about removing the metric names from config props but leaving the metric names in the `StartupTimeMetrics` which would allow configuration by consumers creating their own custom `StartupTimeMetrics` bean instead? I feel like this would probably be a rarely used config option - and one that I would feel better about not encouraging anwyays.

Reply:
I ended up removing the config props. If we want to add them back in, just lmk and I will. This keeps things simpler IMO.